### DISCUSSION TITLE
Add dark mode to default layout

### DISCUSSION THREAD

**Discussion by CodeWithMa** [srid/emanote#403] [Ideas]
Edit the files in https://github.com/srid/emanote/tree/master/default/templates and add dark colors.

I guess the only thing to do is adding `dark:bg-black` etc. in addition to the current colors and tailwind does everything automatically.

See https://tailwindcss.com/docs/dark-mode.

**Comment by mak1A4** [comment]
+1 for this

**Comment by RubikCubed** [comment]
+1, first thing I tried to do was look for dark mode

**Comment by srid** [comment]
This should be fairly simple to do with TailwidCSS. ~~I started a PR long time ago https://github.com/srid/emanote/pull/531 - but somebody wants to pick it up?~~

Implemented as draft PR: https://github.com/srid/emanote/pull/605

**Reply by srid** [reply]
@RubikCubed @mak1A4 @CodeWithMa Can you try this and give feedback? https://github.com/srid/emanote/pull/605

**Reply by srid** [reply]
```
nix run github:srid/emanote/dark
```

Then use the Chrome extension mentioned in docs to toggle it.

**Reply by RubikCubed** [reply]
@srid running it via `nix run` had an error:
```
[Info#ema] ===============================================
[Info#ema] Ema live server RUNNING: http://127.0.0.1:33695 (ws)
[Info#ema] ===============================================
/definitions/mySchema":1:1:
  |
1 | /definitions/mySchema"
  | ^
unexpected '/'
```

The web page also said `Something went wrong`.

But updating the `flake.nix` to use the dark branch, in a project created from emanote-template worked great!

**Reply by RubikCubed** [reply]
I'll give it a more thorough test soon with a bunch more features, just to make sure nothing was missed

**Reply by RubikCubed** [reply]
Only found 1 issue so far, although it's only 404 error, the styling was not readable here:
<img width="679" height="294" alt="image" src="https://github.com/user-attachments/assets/ef82f4f1-fc59-49f8-b042-f374a12a401a" />

**Reply by RubikCubed** [reply]
As well as mermaid diagrams probably needing white arrows:
<img width="439" height="153" alt="image" src="https://github.com/user-attachments/assets/d71ed5c4-45a3-4171-8b74-a5e9e3fb5c5e" />

**Reply by srid** [reply]
@RubikCubed Thanks for checking this. Let's continue our discussion in the PR itself: https://github.com/srid/emanote/pull/605#issuecomment-3292377926

**Comment by srid** [comment]
This is now available: https://emanote.srid.ca/dark-mode

**Reply by srid** [reply]
can also toggle now

<img width="450" height="231" alt="image" src="https://github.com/user-attachments/assets/0a32be4b-85d8-423b-a0fb-869f4ee605f8" />
